src/pages/public_registration/PreviewFile.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import { File } from '../../types/file.types';
+
+const StyledObject = styled.object`
+  width: 100%;
+  height: 25rem;
+`;
+
+const StyledImg = styled.img`
+  max-width: 100%;
+  max-height: 100%;
+`;
+
+interface CommonPreviewProps {
+  url: string;
+}
+
+interface PreviewFileProps extends CommonPreviewProps {
+  file: File;
+}
+
+export const PreviewFile = ({ url, file, ...props }: PreviewFileProps) => {
+  const fileType = file.mimeType.toLowerCase();
+
+  return fileType.includes('pdf') ? (
+    <PreviewPdf url={url} {...props} />
+  ) : fileType.includes('image') ? (
+    <PreviewImg url={url} file={file} {...props} />
+  ) : null;
+};
+
+const PreviewPdf = ({ url, ...props }: CommonPreviewProps) => {
+  const fallbackPdfPreviewUrl = `https://docs.google.com/viewer?url=${url}&embedded=true`;
```

Comment:
Litt usikker på hva som faktisk trengs her, men tror vi må ha noe sånn for at det skal virke om nettleseren ikke har innebygget støtte for PDF-visning

https://stackoverflow.com/a/7044015

Reply:
mulig det uansett er litt dumt å ta inn google-stuff, ref gdpr og personvernerklæringa...

Reply:
jeg vet ikke hva reglene er på dette. men ser litt rart ut at vi skal bruke google docs 🤔

Reply:
enig, tror jeg bare fjerner det. det er bare en fallback, så vi får se på det senere hvis vi trenger noe lignende